### PR TITLE
 📝 Translation of environment-variables.md to pt-br [ci skip]

### PR DESCRIPTION
**Pull request by fplucas**

### CODE REVIEW

File: docs-translations/pt-BR/api/environment-variables.md [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+process.env.GOOGLE_API_KEY = 'YOUR_KEY_HERE'
+```
+
+Para maiores instruções de como conseguir uma chave da API do Google, visite
+[essa página](https://www.chromium.org/developers/how-tos/api-keys).
+
+Por padrão, a mais recente chave da API do Google pode não ser permitida para
+fazer requisições de geolocalização. Para habilitar as requisições, acesse
+
+[essa página](https://console.developers.google.com/apis/api/geolocation/overview).
```
Comment (TiagoDanin) [inline docs-translations/pt-BR/api/environment-variables.md]: Put it together with another paragraph.

File: docs-translations/pt-BR/api/environment-variables.md [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+
+### `ELECTRON_NO_ASAR`
+
+Desabilita o suporte ASAR.Essa variável é suportada somente em processos
```
Comment (TiagoDanin) [inline docs-translations/pt-BR/api/environment-variables.md]: Space between two words

**Comment by zeke** [comment]
Thanks for your contribution!

I'm happy to merge this when it's ready, but I wanted to let you know that we're working on a new system for translating Electron's docs. See the readme at [electron/electron-i18n](https://github.com/electron/electron-i18n) for info on how to get started, or you can jump right into translating at https://crowdin.com/project/electron

**Comment by TiagoDanin** [comment]
Suggestion of titles:
- `Variáveis de ambiente` to `Variáveis de Ambiente`
- `Variáveis de produção` to `Variáveis de Produção`

@fplucas What do you think?

**Comment by fplucas** [comment]
@TiagoDanin gramattically in portuguese it's not right, we normally write just the first character of the phrase in uppercase.

**Comment by zeke** [comment]
Thanks again! Like I said above, please do future translation here: https://crowdin.com/project/electron

Eventually the `/docs-translations` directory will be retired from this repo in favor of this: https://github.com/electron/electron-i18n/tree/master/content

**Pull request merged**